docs/telemetry/CollectionPolicy.md

```diff
@@ -0,0 +1,131 @@
+
+## Metrics collection expectations
+
+It's important to clarify that our intention for telemetry within our projects is solely to prioritize our work efforts in bringing us all to the web3 mecca. This includes not violating our users expectations of privacy nor anonymity.
+
+### Application data vs User data
+
+Our telemetry is solely focused on the collection of application data, not user data.
+
+#### What is Application data?
+
+Application data only contains information about events, modules, performance, and use of our applications, not the users themselves.
+
+#### What is User data?
+
+User data can vary across industries and applications, but is generally data that is used to classify or categorize users, typically for targeting classes or categories of users via campaigns or other marketing methods for various purposes. Think demographics, emails, physical addresses, personal preferences, and marketing.
+
+As a general rule, we do not collect user or personal data. See our [Privacy Policy](./PrivacyPolicy.md) for more details.
+
+### Metric features and their groupings
+
+The metric features in the first column name and their descriptions come directly from [countly's support website](https://support.count.ly/hc/en-us/articles/360037441932-Web-analytics-JavaScript-#features-for-consent).
+
+| Metric feature name | Consent method | Metric feature group name | Metric feature description                                                     | What do we use this metric for?                                                          |
+|:-------------------:|----------------|---------------------------|--------------------------------------------------------------------------------|------------------------------------------------------------------------------------------|
+| sessions            | Opt-out        | Minimal                   | tracks when, how often, and how long users use your website                    | For determining unique daily/weekly/monthly active users in order to prioritize projects |
+| views               | Opt-out        | Minimal                   | allows for the views/pages accessed by a user to be tracked                    | For tracking application pages and component views only                    |
+| events              | Opt-out        | Minimal                   | allows your events to be sent to the server                                    | Timed and one-off application events such as app_start, app_close, function_used, etc. |
+| crashes             | Opt-in         | Performance               | allows JavaScript errors to be tracked                                         | Not used yet - Eventually for bug fixes and debugging |
```

Comment:
That's one good reason. But generally, it was marked as opt-in because I'm trying to keep the amount of opt-out metric groupings small to reduce risk for ourselves and our users. 

I've since switched all the groupings to "opt-out" or "N/A" with a note:

> **Note:** The above groupings are our global settings. You will want to check the project specific `./docs/telemetry/COLLECTED_DATA.md` for variations.